mir-module/src/main/resources/config/mir/mycore.properties

```diff
@@ -371,8 +371,8 @@ MCR.OAIDataProvider.OAI2.Search.RequestHandler=/oai_search
 MCR.OAIDataProvider.OAI2.Search.Restriction=objectType\:mods
 MCR.OAIDataProvider.OAI2.Search.FromUntil=modified
 
-MCR.OAIDataProvider.OAI2.Friends.OpenAgrar=https://openagrar.bmel-forschung.de/servlets/OAIDataProvider
-MCR.OAIDataProvider.OAI2.Friends.RosDok=http://rosdok.uni-rostock.de/oai
+# MCR.OAIDataProvider.OAI2.Friends.OpenAgrar=https://openagrar.bmel-forschung.de/servlets/OAIDataProvider
+# MCR.OAIDataProvider.OAI2.Friends.RosDok=http://rosdok.uni-rostock.de/oai
```

Comment:
use https:// - URL

Reply:
updated the MyCoRe side as well, ... https://github.com/MyCoRe-Org/mycore/pull/1948